macros/src/main/scala/packaged/macros2/package.scala

```diff
@@ -0,0 +1,21 @@
+package packaged
+
+import scala.gestalt._
+
+package object macros2 {
+  inline def plus(a: Int, b: Int): Int = meta {
+    q"$a + $b"
+  }
+
+  object PlusObj {
+    inline def plus(a: Int, b: Int): Int = meta {
+      q"$a + $b"
+    }
+  }
+
+  implicit class PlusFor(val a: Int) {
+    inline def plus(b: Int): Int = meta {
+      q"$this.a + $b"
+    }
+  }
```

Comment:
Thanks for the explanation. Can the macros go into the existing `macros` package? Or maybe we can rename `macro2` to `insidePackage`? That makes clear the intent. I have a worry about having too many different folders that it becomes difficult for contributors to understand the rationale behind them.